mediapipe/tasks/ios/core/sources/MPPTaskRunner.mm

```diff
@@ -59,7 +62,20 @@ - (instancetype)initWithCalculatorGraphConfig:(CalculatorGraphConfig)graphConfig
 }
 
 - (BOOL)sendPacketMap:(const PacketMap &)packetMap error:(NSError **)error {
+  if (!_initializedWithPacketsCallback) {
```

Comment:
This now means that we can initialize a task that will only ever throw errors. Is it possible to revert to the old behavior and throw an error on the initialization?

Reply:
Yeah, but then this whole change is unnecessary right? I mean wouldn't it suffice to just make all the delegate methods @required. Can I close this PR ?

Reply:
Probably. I think we can have `@required` methods and tests that the delegate object is set itself (like I believe we do today).